src/annotations/components/EditAnnotationOverlay.tsx

```diff
@@ -33,15 +33,21 @@ export const EditAnnotationOverlay: FC = () => {
   const dispatch = useDispatch()
   const {clickedAnnotation} = useSelector(getOverlayParams)
 
-  const handleSubmit = (editedAnnotation: EditAnnotation): void => {
+  const getErrorMessage = (err: any): string | any => {
```

Comment:
Not necessary, we can simply import this:

https://github.com/influxdata/ui/blob/master/src/utils/api.ts#L9